app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
||= là gì e nhỉ

Reply:
Dạ nó giống như toán tử gán nhưng nó chỉ gán khi mà chính nó là mang giá trị nill
Trong câu lệnh trên nó sẽ kiểm tra xem @current_user có phải giá trị nil hay không nếu nil thì nó sẽ được gán bằng User.find_by(id: session[:user_id]) còn ko nil thì nó vẫn giữ nguyên giá trị